Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+elifePipeline {
+    def commit
+    DockerImage image
+    stage 'Checkout', {
+        checkout scm
+        commit = elifeGitRevision()
+    }
+
+    node('containers-jenkins-plugin') {
+        stage 'Build images', {
+            checkout scm
+            dockerComposeBuild(commit)
+        }
+
+        stage 'Project tests', {
+            try {
+                sh 'docker-compose run csv-generator true'
```

Comment:
I think here we could run the pytests for both containers.
We could add `project_tests.sh` (or do we now use hyphens) which runs tests across all containers?
We could also add a `project_tests.sh` in each of the projects.
Or both (i.e. `project_tests.sh` at the root would call `project_tests.sh` in each project - via docker-compose run)

Reply:
These are two different projects that do not share code, so go for two different `project_tests.sh` in the two folders, they can be called with separate `docker-compose run` commands. Jenkins can then execute them both, possibly in parallel to save time.